doc/source/user_guide/fields/field_data.rst

```diff
@@ -132,19 +132,19 @@ To obtain pathlines field data, use ``PathlinesFieldDataRequest``:
 .. code-block:: python
 
   >>> from ansys.fluent.core.services.field_data import PathlinesFieldDataRequest
-  >>> pathlines_request_1 = PathlinesFieldDataRequest(field_name="x-velocity", surfaces=["inlet"])
-  >>> path_lines_data = field_data.get_field_data(pathlines_request_1)
+  >>> velocity_pathlines_request = PathlinesFieldDataRequest(field_name="x-velocity", surfaces=["inlet"])
+  >>> velocity_path_lines_data = field_data.get_field(velocity_pathlines_request)
 
   # Vertices shape: (29565, 3) - 29565 pathline points, each with coordinates (x, y, z).
   # Lines: A list where each entry contains indices of vertices forming a pathline.
   # Velocity shape: (29565,) - Scalar velocity values at each pathline point.
-  >>> path_lines_data["inlet"]["vertices"].shape
+  >>> velocity_path_lines_data["inlet"]["vertices"].shape
   (29565, 3)
-  >>> len(path_lines_data["inlet"]["lines"])
+  >>> len(velocity_path_lines_data["inlet"]["lines"])
   29303
-  >>> path_lines_data["inlet"]["x-velocity"].shape
+  >>> velocity_path_lines_data["inlet"]["x-velocity"].shape
   (29565,)
-  >>> path_lines_data["inlet"]["lines"][100]
+  >>> velocity_path_lines_data["inlet"]["lines"][100]
```

Comment:
@prmukherj We shall replace `dict` access subsequently.

Reply:
Yes, that is the plan, right?

Reply:
Yes, just confirming to make sure we're all the same page etc

Reply:
There is a task item created by you for that task. Will do it in a separate PR.

Thank you.